src/main/java/com/banktaglayouts/LayoutGenerator.java

```diff
@@ -10,20 +10,72 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Predicate;
 import java.util.stream.Collectors;
 import net.runelite.client.game.ItemVariationMapping;
+import net.runelite.client.game.RunepouchRune;
 
 @Slf4j
 @RequiredArgsConstructor
 public class LayoutGenerator {
 
     private final BankTagLayoutsPlugin plugin;
 
-    public Layout basicLayout(List<Integer> equippedItems, List<Integer> inventory, Layout currentLayout, int duplicateLimit) {
-        return basicLayout(equippedItems, inventory, Collections.emptyList(), Collections.emptyList(), currentLayout, duplicateLimit);
+    public Layout generateLayout(List<Integer> equippedItems, List<Integer> inventory, List<RunepouchRune> runePouch, Layout currentLayout, int duplicateLimit, boolean alternateLayoutStyle) {
+        if(alternateLayoutStyle) {
+            return prettyLayout(equippedItems, inventory, runePouch, Collections.emptyList(), currentLayout, duplicateLimit);
+        }
+        return flatLayout(equippedItems, inventory, Collections.emptyList(), Collections.emptyList(), currentLayout, duplicateLimit);
+    }
+
+    public Layout prettyLayout(List<Integer> equippedItems, List<Integer> inventory, List<RunepouchRune> runePouch, List<Integer> additionalItems, Layout currentLayout, int duplicateLimit) {
+        Layout previewLayout = Layout.emptyLayout();
+
+        System.out.println(equippedItems);
+        System.out.println(runePouch.stream().map(RunepouchRune::getItemId).collect(Collectors.toList()));
+
+        // lay out equipped items.
+        equippedItems = equippedItems.stream()
+                .map(itemId -> plugin.itemManager.canonicalize(itemId)) // Weight reducing items have different ids when equipped; this fixes that.
+                .collect(Collectors.toList());
+
+        previewLayout.putItem(equippedItems.get(0), EquipmentSlots.HEAD.index);
+        previewLayout.putItem(equippedItems.get(1), EquipmentSlots.CAPE.index);
+        previewLayout.putItem(equippedItems.get(2), EquipmentSlots.NECK.index);
+        previewLayout.putItem(equippedItems.get(13), EquipmentSlots.AMMO.index);
+        previewLayout.putItem(equippedItems.get(3), EquipmentSlots.WEAPON.index);
+        previewLayout.putItem(equippedItems.get(4), EquipmentSlots.CHEST.index);
+        previewLayout.putItem(equippedItems.get(5), EquipmentSlots.OFFHAND.index);
+        previewLayout.putItem(equippedItems.get(7), EquipmentSlots.LEGS.index);
+        previewLayout.putItem(equippedItems.get(9), EquipmentSlots.GLOVES.index);
+        previewLayout.putItem(equippedItems.get(10), EquipmentSlots.BOOTS.index);
+        previewLayout.putItem(equippedItems.get(12), EquipmentSlots.RING.index);
```

Comment:
I think the indexes should be inlined rather than using an enum; the enum isn't used anywhere else.

Reply:
I opted to use an enum because otherwise I would need to comment each line with the slot info (Head, Gloves, Boots, etc) to change it later easily if needed. I can change it to comments with inline indexes if you prefer.

Reply:
I do think comments would be nicer. Having the index's value separate from the one place it's used isn't great imo.

Reply:
oh actually, you could use Runelite's `EquipmentInventorySlot` as indexes into equippedItems; that would document which line is for which slot better than a comment would.